### PR TITLE
Add action plugin for dnos9_template module

### PR DESCRIPTION
**Pull request by skg-net**
##### ISSUE TYPE
- New Module Pull Request
##### COMPONENT NAME
- modules/core/network/dnos9/dnos9_template
##### ANSIBLE VERSION
-  ansible 2.2.0 (devel a1947401ba) last updated 2016/09/08 17:44:28 (GMT -700)
  lib/ansible/modules/core: (detached HEAD db38f0c876) last updated 2016/09/08 17:44:58 (GMT -700)
  lib/ansible/modules/extras: (detached HEAD 8bfdcfcab2) last updated 2016/09/08 17:44:58 (GMT -700)
  config file = /home/skg/lothlorien/ansible.cfg
  configured module search path = Default w/o overrides
##### SUMMARY

Add action plugin for dnos9_template module

### CODE REVIEW

**Comment by skg-net** [comment]
@privateip @gundalow, Please take a look at it.

**Comment by skg-net** [comment]
ready_for_review

File: lib/ansible/plugins/action/dnos9_template.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2015 Peter Sprygada <psprygada@ansible.com>
```
Comment (gundalow) [inline lib/ansible/plugins/action/dnos9_template.py:2]: This still has Peters name on it

**Comment by gundalow** [comment]
needs_revision

**Comment by gundalow** [comment]
Merged, thanks

**Pull request merged**